src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random RANDOM = new Random();
+    private static final String DEFAULT_COLOR = "white";
+    private static final int DEFAULT_RADIUS = 10;
+
+    public Figure getRandomFigure() {
+        int figurType = RANDOM.nextInt(5);
+        String color = new ColorSupplier().getRandomColor();

```

Comment:
You should create several random Figures, so you will use a loop. Please don't create a 'new ColorSupplier()' inside the loop or a method that is called in a loop. It's better to create an instance of 'ColorSupplier' once and reuse it.